feature-libs/checkout/scheduled-replenishment/components/checkout-schedule-replenishment-order/checkout-schedule-replenishment-order.component.html

```diff
@@ -50,11 +50,36 @@
   >
     <div class="cx-replenishment-form-data-container">
       <div *ngIf="!isMonthly" class="cx-days">
-        <span class="form-data-label">{{
-          'checkoutScheduledReplenishment.every' | cxTranslate
-        }}</span>
+        <span
+          class="form-data-label"
+          *cxFeature="'!a11yScheduleReplenishment'"
+          >{{ 'checkoutScheduledReplenishment.every' | cxTranslate }}</span
+        >
+        <label
+          class="form-data-label"
+          *cxFeature="'a11yScheduleReplenishment'"
+          for="order-replenishment-period-type"
+          >{{ 'checkoutScheduledReplenishment.every' | cxTranslate }}</label
+        >
         <ng-container *ngIf="isWeekly; else isDaily">
           <select
+            *cxFeature="'!a11yScheduleReplenishment'"
+            class="form-control"
+            (change)="changeNumberOfWeeks($event.target.value)"
+          >
+            <option
+              *ngFor="let nWeeks of numberOfWeeks"
+              [value]="nWeeks"
+              [selected]="
+                nWeeks === scheduleReplenishmentFormData.numberOfWeeks
+              "
+            >
+              {{ nWeeks }}
+            </option>
+          </select>
+          <select
+            *cxFeature="'a11yScheduleReplenishment'"
+            id="order-replenishment-period-type"
```

Comment:
We can introduce `id` attributes without a feature toggle because its not a breaking change.

Reply:
I've just updated it.